CMS/templates/contentPages/resources_page.html

```diff
@@ -86,24 +86,32 @@ <h1>{{ page.site_heading }}</h1>
 
                         <div id="search-results" class="spacer-top-1">
                             <div class="row">
-                                <div class="crc-thumbnails">
-                                    <div id="internal-resources">
-                                      <div id="popular">
-                                          {% for resource_page in page.resource_list %}
+                                    <ul id="internal-resources" class="crc-thumbnails">
```

Comment:
This page appears to be using the `<div>` version - has the report called it out as not correct?